package.json

```diff
@@ -380,9 +380,14 @@
     "@microsoft/api-extractor/typescript": "5.3.3",
     "swc-loader": "^0.2.6",
     "prettier": "2.8.8",
-    "puppeteer": "19.6.0",
+    "puppeteer": "19.6.3",
     "ws": "8.17.1",
-    "**/@mdx-js/react/@types/react": "^17"
+    "**/@mdx-js/react/@types/react": "^17",
+    "**/prismjs": "^1.30.0",
```

Comment:
react-syntax-highlighter dependency `refractor` pins prismjs to `~` which makes it impossible to upgrade prismjs.